the-collab-lab/tcl-22-smart-shopping-list
# Implement Material-UI and new folder structure for AddItems.js

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

1. Implement new folder structure;
2. Use Material UI components for `AddItems.js`:
* TextField
* FormControl
* FormLabel
* FormControlLabel
* RadioGroup
* Radio
* Button
***Proper styles coming soon***

## Related Issue

#14 

## Acceptance Criteria

- As a user, I want the app to look professional and welcoming

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓  | :hammer: Refactoring       |

## Updates

### Before

![screencapture-tcl-22-shopping-list-pr47-cf-material-ui-welco-1fukodsg-web-app-additems-2021-05-19-18_24_30](https://user-images.githubusercontent.com/68360696/118892834-7f650500-b8cf-11eb-919f-784e4353eb74.png)


### After

![screencapture-localhost-3000-additems-2021-05-19-20_55_04](https://user-images.githubusercontent.com/68360696/118902866-8a766000-b8e4-11eb-91be-ea9613e5d1a4.png)


## Testing Steps / QA Criteria

To test the feature, use one of the following ways:

1. Follow the [preview link](https://tcl-22-shopping-list--pr50-dt-mui-add-items-ddp8uu0t.web.app)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin dt-mui-add-items` and check that branch out with `git checkout origin dt-mui-add-items`
- Run npm start to launch the app in your browser at http://localhost:3000

In the app, navigate to Add Groceries view to see the changes.
